### DISCUSSION TITLE
brightness change service?

### DISCUSSION THREAD

**Discussion by belkone** [RavenSystem/esp-homekit-devices#1610] [General]
Hi!
Is it possible to add 'brightness change' characteristic to a light bulb service?
Other devices have this option, is it also possible in HAA? I need this to make the dimmer.

haa device:
![IMG_6452](https://user-images.githubusercontent.com/14234849/140426232-effff69d-ab2c-4eb1-9ebe-330268d1fbdd.PNG)

other device (from homebridge-hue):
![IMG_6451](https://user-images.githubusercontent.com/14234849/140426219-8a4938d7-5406-4875-bd4b-0f1c690a3eca.PNG)

used json config: `{"c":{"q":600,"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[13,5],"b":[{"g":0}]}]}`

**Comment by belkone** [comment]
maybe anyone has any idea how else to dimming a light bulb with an external button?

**Reply by RavenSystem** [reply]
When light is on, do a double press and light becomes to change brightness. To fix it, do a single press.

**Comment by RavenSystem** [comment]
I don't know what is 'brightness change' characteristic and how it can do "dimmer".
